### ISSUE TITLE
앞으로의 작업 진행사항에 대하여..

### ISSUE THREAD

**Issue by ljhh-0611** [16-2-SKKU-OSS/16-2-OSS-C#6]
지난번 회의의 결과로

- 하둡 발표팀: 하둡 프레젠테이션을 md파일로 옮긴다.

- 아파치 라이센스 발표팀: 마찬가지로 아파치 라이센스를 md 파일로 옮긴다.

- 깃허브 발표팀: tutorial을 먼저 번역한다. - 첫번째 튜토리얼을 끝낸다.

이렇게 진행하기로 했습니다!

우선은, Milestone에 있는 것 처럼 화요일까지 위의 사항을 끝내는 것을 목표로 잡아봤습니다.

화이팅!

**Comment by ljhh-0611** [comment]
아 그리고 서로의 소통을 위해서 읽으셨을때 아무말대잔치라도 코멘트 주시면 감사드리겠습니당~

**Comment by YunJeon** [comment]
번역 열심히 하겠습니다~:)
아무말대잔치ㅎㅎ

**Comment by sdkjd** [comment]
저도 열심히 하겠나이다. ㅎㅎ

**Comment by YoungAndDo** [comment]
'아무말대잔치' - **오늘도 춥네요**

**Comment by gw4621** [comment]
알겠습니담

**Comment by ljhh-0611** [comment]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ 아니 그렇다고 진짜 말그대로 아무말을 치시다닠ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ  각자 자기가 들어간 작업에 대해 이슈를 올려서 작업을 나누시면 되시겠습니다!!

**Comment by kds135** [comment]
제가 하둡 발표팀인데
하둡 발표팀: 하둡 프레젠테이션을 md파일로 옮긴다
이걸 하면 되는건가요???

**Comment by lkiung** [comment]
Hadoop 발표팀은 md file로 구현하기 보다 wiki에 내용을 구현하는게 낫지않나여??

**Comment by YunJeon** [comment]
근데 저희 원래 회의할 때, 먼저 md file로 구현하고, 정리해서 wiki에 옮기기로 하지 않았나요??
저는 그렇게 알고 있는데...

**Comment by ljhh-0611** [comment]
예 저희 무조건 일단 md파일로 계속 commit작업을 진행해야 history에 남고 그걸 바탕으로 바꿀 부분을 바꾸기 편하기때문에
일단 project에서 작업하고 완성되면 wiki에 그대로 넣는 방식으로 하기로 했습니다!

2016. 11. 30. 오후 6:58에 "YunJeon" <notifications@github.com>님이 작성:

**Comment by YoungAndDo** [comment]
저희 md파일에 넣을 이미지는 Github에 넣어서 링크로 접근이 안되는 것 같은데 혹시 되나요?

만약 안된다면 웹서버 하나에 이미지 넣고 링크로 불러오는 방법도 있는데 이건 어떤가요?

**Comment by lkiung** [comment]
깃헙에 넣어서 이미지 되여!! ![이름](링크)로 하시면 되여!!

**Comment by YoungAndDo** [comment]
아하! 확인해보니깐 링크를 주의해서 걸어야했네요 ㅎㅎ 감사합니다.

그런데 혹시 위키에 넣을 그림들 각각 항목에 맞게 폴더에 집어넣으면 어떨까요오오?

**Comment by kds135** [comment]
헛 저도 좀 늦게나마 했지만, wiki에 하둡 발표파일 올렸습니다. 이거 md파일에도 올리면 되나요???

**Comment by kds135** [comment]
md파일에도 하둡 발표 제 부분 자료 올렸습니다

**Comment by sdkjd** [comment]
오늘까지 wiki에 아파치 올리겠습니다.

**Comment by sdkjd** [comment]
일단 업로드 했는데 수정할 사항있으시면 말씀해주십시요
